api/serializers/team.py

```diff
@@ -33,3 +33,50 @@ def create(self, validated_data):
     def update(self, instance, validated_data):
         validated_data['status'] = Status.objects.get(name=check_team(validated_data.get('users'), team=instance))
         return super().update(instance, validated_data)
+
+
+class AddTeamMemberSerializer(serializers.Serializer):
+    team = serializers.PrimaryKeyRelatedField(queryset=Team.objects.all())
+    member = serializers.PrimaryKeyRelatedField(queryset=User.objects.all())
+
+    def validate(self, attrs):
+        team = attrs.get('team')
+        member = attrs.get('member')
+        if member.detail.type.name in [UserTypeChoices.WORKER, UserTypeChoices.MANAGER]:
```

Comment:
I think you can add also: mechanics here
And question: Will it be improved? Now, we have only worker/manager/mechanic adding, but it also has: IT department, supervisor.

Reply:
Add checking count of manager: it should has only 1 manager